### PR TITLE
Build pagination feature in two alternative ways

### PR DESCRIPTION
**Pull request by stepris**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -12,8 +12,9 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+const maxPage = 42;
+let page = 41;
```
Comment (EmmaDir) [inline index.js]: Page should start with 1

File: index.js [hunk 2]
```diff
@@ -43,3 +50,45 @@ async function fetchCharacters() {
 }
 
 fetchCharacters();
+
+// Alternative way to write the pagination
```
Comment (EmmaDir) [inline index.js:54]: Alternative way looks better

File: index.js [hunk 3]
```diff
@@ -43,3 +50,45 @@ async function fetchCharacters() {
...
+paginationUpdate();
+
+nextButton.addEventListener("click", () => {
+  paginationUpdate(1);
+});
+prevButton.addEventListener("click", () => {
+  paginationUpdate(-1);
+});
+
+// Simple way to write pagination
```
Comment (EmmaDir) [inline index.js:75]: Worked already

**Review by EmmaDir — CHANGES_REQUESTED** [review]
Two additional comments

**Review by EmmaDir — APPROVED** [review]
Code looks good!

**Review by Abvali — APPROVED** [review]
its OK

**Review by stekug — APPROVED** [review]
Habe auch nochmal drauf geschaut, sieht super aus!

**Pull request merged**